api/lib/domain/read-models/JuryCertificationSummary.js

```diff
@@ -2,6 +2,16 @@ const { status: assessmentResultStatuses } = require('../models/AssessmentResult
 
 const STARTED = 'started';
 
+const ACQUIRED = true;
+const REJECTED = false;
+const NOT_PASSED = null;
+
+const partnerCertificationStatusToDisplayName = {
```

Comment:
Le choix du wording à afficher à l'utilisateur devrait être fait coté front.
Ici coté back on pourrait renvoyer une clé (`validated`, `rejected`, `not-passed`)

Reply:
Yes, en plus ici on aurait une clé null (NOT_PASSED) qui aurait pour valeur 'non testée', c'est pas top